src/sentry/tasks/unmerge.py

```diff
@@ -31,8 +39,6 @@ def fetch(*key):
 
 
 def get_caches():
-    from sentry.models import Environment, GroupRelease, Project, Release
-
     return {
         'Environment': cache(
```

Comment:
These could be modified back to direct model class references, I don't have a strong preference on whether that change should be made or not.